SwissTransferFeatures/TransferDetailsView/PasswordPanelView.swift

```diff
@@ -0,0 +1,83 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakCoreSwiftUI
+import STResources
+import SwiftUI
+import SwissTransferCoreUI
+
+struct PasswordPanelView: View {
+    @Environment(\.dismiss) private var dismiss
+
+    @State private var isShowingPassword = false
+
+    let password: String
+
+    private var passwordValue: String {
+        isShowingPassword ? password : String(repeating: "*", count: password.count)
+    }
+
+    var body: some View {
+        VStack(spacing: IKPadding.large) {
+            Text(STResourcesStrings.Localizable.sharePasswordTitle)
+                .font(.ST.headline)
+                .foregroundStyle(Color.ST.textPrimary)
+
+            Text(STResourcesStrings.Localizable.sharePasswordDescription)
+                .font(.ST.body)
+                .foregroundStyle(Color.ST.textSecondary)
+                .multilineTextAlignment(.center)
+
+            HStack {
+                Text(passwordValue)
+                    .frame(maxWidth: .infinity, alignment: .leading)
+
+                Button {
+                    withAnimation {
+                        isShowingPassword.toggle()
+                    }
+                } label: {
+                    isShowingPassword ? STResourcesAsset.Images.eyeSlash.swiftUIImage : STResourcesAsset.Images.eye.swiftUIImage
+                }
+            }
+            .foregroundStyle(Color.ST.textSecondary)
+            .padding(value: .intermediate)
+            .overlay(Color.ST.textFieldBorder, in: .rect(cornerRadius: IKRadius.small).stroke())
+            .padding(.bottom, 30)
+
+            CopyToClipboardButton(item: password, labelStyle: .ikLabel)
```

Comment:
For the bottom sheets of this PR, you can wraps buttons around `BottomButtonsView` 👍 
Then you should configure the floatingPanel with this: `bottomPadding: .zero`